custom_components/sat/climate.py

```
@@ -494,44 +503,52 @@ def hvac_action(self):
-        return max(self.errors, key=lambda error: error.value)
-
     @property
     def setpoint(self) -> float | None:
         return self._setpoint
 
     @property
     def requested_setpoint(self) -> float:
-        """Get the requested setpoint based on the heating curve and PID output."""
+        """Get the requested setpoint based on the heating curve and PIDs."""
         if self.heating_curve.value is None:
             return MINIMUM_SETPOINT
```

Ruff (0.14.8)

`MINIMUM_SETPOINT` may be undefined, or defined from star imports

(F405)